assets/js/atomic/blocks/product-elements/image/test/block.test.js

```diff
@@ -56,7 +56,7 @@ describe( 'Product Image Block', () => {
 		test( 'should render an anchor with the product image', () => {
 			const component = render(
 				<ProductDataContextProvider product={ productWithImages }>
-					<Block productLink />
+					<Block showProductLink />
```

Comment:
Not introduced in this PR, but suggesting it here in case you want to take the chance and improve it. :slightly_smiling_face:  We always try to be explicit about prop values. In this case, it would be:
```suggestion
					<Block showProductLink={ true } />
```
This makes the code easier to understand, because it's easy to get confused by the way React and HTML treat omitted prop/attribute values and the way JavaScript treats object properties. Let me explain: React props (and HTML attributes) without an explicit value default to true (like in this case), while JS object properties default to a variable with the same name (ie: `{ showProductLink }` = `{ showProductLink: showProductLink }`).

Reply:
Thanks for your review and pointing out to mention the props value explicitly. I applied your suggested change and merged the PR accordingly.